exopy_pulses/tasks/tasks/instrs/views/transfer_sequence_task_view.enaml

```diff
@@ -287,9 +329,9 @@ enamldef TransferPulseSequenceView(InstrTaskView): view:
         if change['value']:
             sequence = change['value']
             sequence.observe('context', _check_context_driver_compatibility)
-            if sequence.context:
-                validate_context_driver_pair(view.root.core,
-                                             task.sequence.context, task, view)
+            #if sequence.context:
```

Comment:
Why is it commented out ?

Reply:
No idea but according to my colleagues who use the code, it works without this line

Reply:
Obviously commenting out checks does not cause crashes right away but this is still suspicious and I would really like to understand why it was deemed necessary.